README.md

```diff
@@ -86,3 +86,21 @@ Depending on the task, docker memory for an ARMORY container must be at least 8
 On Mac and Windows Desktop versions, this defaults to 2 GB. See the docs to change this:
 * [Mac](https://docs.docker.com/docker-for-mac/)
 * [Windows](https://docs.docker.com/docker-for-windows/)
+
+### Docker Cleanup
+Subsequent versions of ARMORY will download new Docker images, but will not clean up existing ones.
+To display the set of current images:
+```
+docker images
+```
+To delete images, see the docs for [docker rmi](https://docs.docker.com/engine/reference/commandline/rmi/)
+or this [StackOverflow post](https://stackoverflow.com/questions/44785585/how-to-delete-all-docker-local-docker-images).
```

Comment:
Not sure we want to link to how to delete all images. A safer call would be:
`docker rmi $(docker images | grep 'twosixarmory')`

Reply:
Would that delete the intermediate images as well? If not, then maybe we should just reference the docker rmi.

Reply:
Not positive. Lets just reference docker rmi for now. We can handle this for the user with #174